concourse/pipelines/ci.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       trigger: true
     - file: govuk-infrastructure/concourse/pipelines/ci.yml
       set_pipeline: ci-govuk-infrastructure-main
+      var_files:
+        - govuk-infrastructure/concourse/parameters/ci/ci.yml
```

Comment:
I wonder if this file should go into the `test` environment params? Seems a bit weird to have the ci pipeline be a psuedo 'ci' environment rather than explicitly a part of the test env.

Reply:
I think we should keep it because it is consistent with the CI concourse team name